app/controllers/trainer_controller.rb

```diff
@@ -3,12 +3,24 @@ class TrainerController < ApplicationController
 
   def review_card
     @card = current_user.cards.find(params[:card_id])
-    if @card.check_translation(trainer_params[:user_translation])
-      flash[:notice] = 'Вы ввели верный перевод. Продолжайте.'
-      redirect_to root_path
-    else
-      flash[:alert] = 'Вы ввели не верный перевод. Повторите попытку.'
-      redirect_to root_path(id: @card.id)
+
+    levenshtein_distance =
+        @card.check_translation(trainer_params[:user_translation])
+
+    case levenshtein_distance
+      when 0
+        flash[:notice] = 'Вы ввели верный перевод. Продолжайте.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.